setup.py

```diff
@@ -25,7 +26,7 @@
 # Used for the long_description.  It's nice, because now 1) we have a top level
 # README file and 2) it's easier to type in the README file than to put a raw
 # string in below ...
-def read(fname):
+def read(fname: Any) -> Any:
     return open(os.path.join(os.path.dirname(__file__), fname)).read()
```

Comment:
Unfortunately, declaring `Any` doesn't help us catch invalid values. For example, this function would throw an exception if passed an integer, but mypy wouldn't warn us ahead of time since everything matches `Any`.

What do you think these types should be?

Reply:
I was thinking of `str` but we also have `bytes` objects representing a path, is it possible to combine `str` and `bytes`